### PR TITLE
Add a default trie-memory-tracker feature to the cli

### PR DESCRIPTION
**Pull request by expenses**
This is essentially a companion PR to https://github.com/paritytech/substrate/pull/6745.

### CODE REVIEW

**Comment by expenses** [comment]
bot merge

File: cli/Cargo.toml [hunk 1]
```diff
@@ -34,12 +34,13 @@ sc-service = { git = "https://github.com/paritytech/substrate", branch = "master
 wasm-bindgen = { version = "0.2.57", optional = true }
 wasm-bindgen-futures = { version = "0.4.7", optional = true }
 browser-utils = { package = "substrate-browser-utils", git = "https://github.com/paritytech/substrate", branch = "master", optional = true }
+sp-trie = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
```
Comment (ordian) [inline cli/Cargo.toml:37]:
```suggestion
# this crate is used only to enable `trie-memory-tracker` feature
# see https://github.com/paritytech/substrate/pull/6745
sp-trie = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
```

**Comment by expenses** [comment]
bot merge

File: cli/Cargo.toml (hunk 1 above)
Comment (expenses) [reply cli/Cargo.toml:39]: Good idea!

**Pull request merged**